OWNERS

```diff
@@ -5,10 +5,11 @@ approvers:
   - saikat-royc
   - leiyiz
 reviewers:
```

Comment:
I think you can also remove saad and jing as they are no longer active in this repo. We can always add them back in the future. It'll help with bot assignments

Reply:
Ack